apps/contributions/models.py

```diff
@@ -343,5 +345,13 @@ def handle_thank_you_email(self, contribution_received_at=None):
                     "contribution_interval_display_value": self.interval if self.interval != "one_time" else None,
                     "copyright_year": contribution_received_at.year,
                     "org_name": self.revenue_program.organization.name,
+                    # TODO: Missing field to be added on DEV-2892
+                    # "first_name": '',
+                    # "last_name": '',
```

Comment:
[As it turns out,](https://news-revenue-hub.atlassian.net/browse/DEV-2892?focusedCommentId=64466) we'll be getting full name from the Stripe Customer name field, so let's rename to a single field called `contributor_name`:

```suggestion
                    # "contributor_name": '',
```